### ISSUE TITLE
Linting fails with infinite loop

### ISSUE THREAD

**Issue by gabaum10** [vuetifyjs/eslint-plugin-vuetify#5]
I was hoping the next few beta releases would resolve this, however beta.3 doesn't seem to do it.  When I add the plugin to my eslint config, I'm getting a loop of the following error until it fails:

```
(node:18696) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 VElement listeners added. Use emitter.setMaxListeners() to increase limit
error Command failed with exit code 1.
```

the eslintrc.js file looks like this:

```
// https://eslint.org/docs/user-guide/configuring

module.exports = {
  root: true,
  parserOptions: {
    parser: 'babel-eslint'
  },
  env: {
    browser: true,
    jquery: true
  },
  globals: {
    google: true
  },
  extends: [
    // https://github.com/vuejs/eslint-plugin-vue#priority-a-essential-error-prevention
    // consider switching to `plugin:vue/strongly-recommended` or `plugin:vue/recommended` for stricter rules.
    'plugin:vue/recommended',
    // https://github.com/standard/standard/blob/master/docs/RULES-en.md
    'standard'
  ],
  // required to lint *.vue files
  plugins: [
    'vue',
    'vuetify'
  ],
  // add your custom rules here
  rules: {
    // vuetify helper
    'vuetify/no-deprecated-classes': 'error',
    // allow async-await
    'generator-star-spacing': 'off',
    // allow debugger during development
    'no-debugger': process.env.NODE_ENV === 'production' ? 'error' : 'off',
    "indent": 0,
    'curly': ['error', 'all'],

    // vue settings
    'vue/no-v-html': 0,
    'vue/require-default-prop': ['error'],
    'vue/script-indent':[
      "error",
      2,
      {
        "baseIndent": 1,
        "switchCase": 1,
      }
    ],
    'vue/html-indent':[
      "error",
      2
    ],
    "vue/singleline-html-element-content-newline": ["error", {
      "ignoreWhenNoAttributes": true,
      "ignoreWhenEmpty": true
    }],
    "vue/component-name-in-template-casing": ["error", "kebab-case"],
    "vue/html-closing-bracket-newline": ["error", {
      "singleline": "never",
      "multiline": "always"
    }],
    "vue/max-attributes-per-line": ["error", {
      "singleline": 1,
      "multiline": {
        "max": 1,
        "allowFirstLine": false
      }
    }],
    "vue/attributes-order": ["error", {
      "order": [
        "DEFINITION",
        "LIST_RENDERING",
        "CONDITIONALS",
        "RENDER_MODIFIERS",
        "GLOBAL",
        "UNIQUE",
        "TWO_WAY_BINDING",
        "OTHER_DIRECTIVES",
        "OTHER_ATTR",
        "EVENTS",
        "CONTENT"
      ]
    }],
    "vue/no-multi-spaces": ["error", {
      "ignoreProperties": false
    }],
    "vue/order-in-components": ["error", {
      "order": [
        "el",
        "name",
        "parent",
        "functional",
        ["delimiters", "comments"],
        ["components", "directives", "filters"],
        "extends",
        "mixins",
        "inheritAttrs",
        "model",
        ["props", "propsData"],
        "data",
        "computed",
        "watch",
        "LIFECYCLE_HOOKS",
        "methods",
        ["template", "render"],
        "renderError"
      ]
    }],
    "vue/multiline-html-element-content-newline": ["error"],
    "vue/html-closing-bracket-spacing": ["error"]
  }
}
```

Are one of the other rules interfering with it?

**Comment by gabaum10** [comment]
I should mention, this is a massive project with hundreds of components and supporting JS files. Is there a legit memory leak somewhere in the plugin code?

**Comment by KaelWD** [comment]
> Possible EventEmitter memory leak detected. 11 VElement listeners added

https://github.com/vuejs/eslint-plugin-vue/pull/895

> Is there a legit memory leak somewhere in the plugin code?

It's possible, but that message doesn't necessarily indicate it. It's used in the Vuetify monorepo with >800 components without a problem. 
Can you try installing `vue-eslint-parser@^6.0.4` and running `TIMING=1 yarn lint`?

Oh yeah I'm assuming you're using the latest versions of eslint and eslint-plugin-vue too ofc.

**Comment by gabaum10** [comment]
@KaelWD alrigth so I upgraded that stuff and ran the `TIMING=1 yarn lint`.  No luck.  It's still throwing the same error.  

Here is the output:
```
.
.
.
(node:31367) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 VElement listeners added. Use emitter.setMaxListeners() to increase limit
(node:31367) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 VElement listeners added. Use emitter.setMaxListeners() to increase limit
Rule                         | Time (ms) | Relative
:----------------------------|----------:|--------:
import/no-duplicates         |  2532.719 |    28.6%
vue/no-duplicate-attributes  |  1753.095 |    19.8%
vue/script-indent            |   740.195 |     8.4%
no-duplicate-case            |   548.585 |     6.2%
no-unused-vars               |   243.507 |     2.8%
keyword-spacing              |   137.541 |     1.6%
no-extra-parens              |   129.947 |     1.5%
key-spacing                  |   125.033 |     1.4%
no-unmodified-loop-condition |   103.327 |     1.2%
camelcase                    |    95.144 |     1.1%
error Command failed with exit code 1.
```

**Comment by gabaum10** [comment]
What's interesting is the lint rules actually do get checked and correctly error, way up in the terminal output.  I output it to a file and I can see all them.  However, when I try to run the command with the --fix command, the JS heap overflows.

[output.txt](https://github.com/vuetifyjs/eslint-plugin-vuetify/files/3426537/output.txt)

**Comment by gabaum10** [comment]
Looks potentially related to this?  https://github.com/vuejs/eslint-plugin-vue/issues/929

**Comment by gabaum10** [comment]
Question: what version of Node are you running?

**Comment by gabaum10** [comment]
Confirmed:  Those are just noise. The OOM failure was because I needed to allot some more to the node runtime.  I guess including the vuetify eslint plugin pushed it over the edge.  Once I did that, the `--fix` worked and it went along its merry way.  I think your assessment is correct in the `eslint-plugin-vue` is using the incorrect version of `vue-eslint-parser`.

**Comment by KaelWD** [comment]
Ah righto that's a bit of a mess. I can run a memory profile and see if I'm doing something dumb to cause the OOM.

**Comment by 185driver** [comment]
I think I'm experiencing this issue too, though my error message is slightly different:
```bash
yarn lint
yarn run v1.16.0
$ vue-cli-service lint
 DONE  No lint errors found!
(node:5960) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 VElement listeners added. Use emitter.setMaxListeners() to increase limit
(displayed x 12 times)
Done in 6.72s.
```
There's no error when `eslint-plugin-vuetify` is disabled. I'll plan to wait for an update to `eslint-plugin-vue` to see if that resolves it.

**Comment by KaelWD** [comment]
You can safely ignore that one, it doesn't affect anything.

**Comment by annymosse** [comment]
as i tried to use this plugin i got it's impossible to upgrade from 1.5.x to 2.x, as a result of that im doing all the upgrade manually , coz i'm getting error after error even i left my project a weeks to resolve that  problem , its much better to take these weeks to do it manually and save a time.

**Comment by Ruinevo** [comment]
i have a similar problem.

If i delete 'vue/html-indent': ['error', 2] - the problem is being solved

**Comment by KaelWD** [comment]
Again, the MaxListenersExceededWarning can be safely ignored.
